.github/workflows/build_and_test.yml

```diff
@@ -24,45 +39,61 @@ jobs:
       - name: Run ast tests
         run: JDK_9="$JAVA_HOME" ./gradlew :python:ast:test
         if: always()
       - name: Build
         run: JDK_9="$JAVA_HOME" ./gradlew dist
-
       - name: Compile python.kt to Python
-        run: dist/kotlinc/bin/kotlinc-py -libraries dist/kotlinc/lib/kotlin-stdlib-js.jar -Xir-produce-js -output out_ir.py python/experiments/python.kt
+        run: |
+          dist/kotlinc/bin/kotlinc-py \
+          -libraries dist/kotlinc/lib/kotlin-stdlib-js.jar \
+          -Xir-produce-js \
+          -output out_ir.py \
+          python/experiments/python.kt
         if: always()
       - name: Compare out_ir.py
         run: diff -u out_ir.py python/experiments/generated/out_ir.py
         if: always()
-
       - name: Compile python.kt to JavaScript
-        run: dist/kotlinc/bin/kotlinc-js -libraries dist/kotlinc/lib/kotlin-stdlib-js.jar -Xir-produce-js -Xir-property-lazy-initialization -output out-ir.js python/experiments/python.kt
+        run: |
+          dist/kotlinc/bin/kotlinc-js \
+          -libraries dist/kotlinc/lib/kotlin-stdlib-js.jar \
+          -Xir-produce-js \
+          -Xir-property-lazy-initialization \
+          -output out-ir.js \
+          python/experiments/python.kt
         if: always()
       - name: Compare out-ir.js
         run: diff -u out-ir.js python/experiments/generated/out-ir.js
         if: always()
-
       - name: Generate stats about missing IR mapping
-        run: less python/experiments/generated/out_ir.py | grep -Po "visit[a-zA-Z0-9_]+" | sort | uniq -c | sort -nr > missing.txt
+        run: |
+          less python/experiments/generated/out_ir.py | \
+          grep -Po "visit[a-zA-Z0-9_]+" | \
+          sort | \
+          uniq -c | \
+          sort -nr \
+          > missing.txt
         if: always()
       - name: Compare missing.txt
         run: diff -u missing.txt python/experiments/generated/missing.txt
         if: always()
-
       - name: Run end-to-end tests
         run: JDK_9="$JAVA_HOME" python/e2e-tests/run.sh
-        if: ${{ matrix.testTask == 'pythonTest' }}  # TODO: run e2e tests for MicroPython too (#85)
-
+        if: ${{ matrix.testTask == 'pythonTest' }}
       - name: Run box tests (succeed even if they fail)
         run: JDK_9="$JAVA_HOME" ./gradlew :python:box.tests:${{ matrix.testTask }} || true
         if: always()
       - name: Generate box tests reports
-        run: python/experiments/generate-box-tests-reports.main.kts --test-task=${{ matrix.testTask }} --failed-tests-report-path=failed-tests.txt --box-tests-report-path=box-tests-report.tsv --failure-count-report-path=failure-count.tsv
+        run: |
+          python/experiments/generate-box-tests-reports.main.kts \
+          --test-task=${{ matrix.testTask }} \
+          --failed-tests-report-path=failed-tests.txt \
+          --box-tests-report-path=box-tests-report.tsv \
+          --failure-count-report-path=failure-count.tsv
```

Comment:
Well, I was OK with long lines ;)

Reply:
I actually found them easy to work with. Horizontal scrolling is painful :)